### ISSUE TITLE
Feature Request - Alerts for subs, follows and cheers

### ISSUE THREAD

**Issue by Fratm** [relamptk/vr-streaming-overlay#3]
Alerts in chat for subs, follows and cheers, with special highlighting to make them more visible.

**Comment by beniwtv** [comment]
Yes, that is planned, but will be a separate widget you can add to the overlay :)
I'm thinking to also add a stats widget - showing current viewers, stream health, etc... (depending on what the Twitch API can do)
